src/test/java/space/obminyashka/items_exchange/rest/AdvertisementControllerIntegrationTest.java

```diff
@@ -102,6 +105,26 @@ void createAdvertisement_shouldReturn400WhenBlankFieldsButWithoutBlankWish(Adver
                 .hasMessageNotContaining(blankWishesToExchangeMessage);
     }
 
+    @Test
+    @WithMockUser(username = "admin")
+    void createAdvertisement_shouldReturn400WhenInvalidSizeForCategory() throws Exception {
+        final var dto = AdvertisementModificationDtoProducer.createNonExistAdvertisementModificationDto();
+        dto.setSize("invalidSize");
+
+        final var dtoJson = new MockMultipartFile("dto", "json", MediaType.APPLICATION_JSON_VALUE,
```

Comment:
The same object is used in the test above and it'd more useful to extract that variable on the class'es level instead of duplication it.

Reply:
@rpkyrych these are not exactly the same objects, each of them accepts a different DTO.
I can make an additional method that takes this DTO and converts it to a MockMultipartFile